### ISSUE TITLE
GQL Parser = NULL

### ISSUE THREAD

**Issue by iamacarpet** [tomwalder/php-gds#123]
Hello,

I'm trying to do a SELECT query in GQL with a WHERE filter of field = NULL.

It works on the live version of GQL on AppEngine, but it don't work locally using this parser.

I don't mind implementing this if someone can point me in the right direction please?

Regards,
iamacarpet

**Comment by tomwalder** [comment]
This maybe resolved with #164

**Comment by lifeofguenter** [comment]
Yes, this is resolved with #164. You will not be able to directly query like this:

* `... WHERE field IS NULL`
* `... WHERE field = NULL`

But you'll be able to query for `NULL` via parameter binding as seen here: https://github.com/tomwalder/php-gds/pull/164/files#diff-796607b903aea97a7e1d0226ce4cbf55R810 - e.g.:

`... WHERE field = @null` and then pass through `'null' = null`
